src/grt/grt-wave_options-parse-debug.adb

```diff
@@ -0,0 +1,58 @@
+--  GHDL Run Time (GRT) - mono-thread version.
+--  Copyright (C) 2005 - 2014 Tristan Gingold
```

Comment:
On 23/07/16 08:50, Jonsba wrote:
> In src/grt/grt-wave_options-parse-debug.adb
> <https://github.com/tgingold/ghdl/pull/121#discussion_r71968540>:
>
>> @@ -0,0 +1,58 @@
>> +--  GHDL Run Time (GRT) - mono-thread version.
>> +--  Copyright (C) 2005 - 2014 Tristan Gingold
>
> Do you mean that I should write "Copyright (C) 2016 Jonas Baggett" ?

Yes, as you have written it.

> What do I need to change in the first line ?

Add a brief description after the '-', like Debug for Wave_Options parser.